app/views/eventDetails/regOptions.html

```diff
@@ -34,6 +34,17 @@ <h2 class="page-title border" translate>Registration Options</h2>
          popover-append-to-body="true">
       </i>
     </div>
+
+    <div class="checkbox">
+      <label><input type="checkbox" ng-model="conference.combineSpouseRegistrations">
+        <translate>Combine spouse registrations</translate>
+      </label>
+      <i class="question-popover fa fa-question-circle"
```

Comment:
@canac I think we should add something here explicitly communicating this field doesn't do anything unless the spouse is on a family registration.